tests/utils.js

```diff
@@ -302,7 +302,17 @@ const setUserContactDoc = (attempt=0) => {
     });
 };
 
+/**
+ * 
+ * @param {array} except - exeptions in the delete method. If this parameter is empty 
+ *                         everything will be deleted from the config, including all the enketo forms.
+ * @param {boolean} ignoreRefresh 
+ */
 const revertDb = async (except, ignoreRefresh) => {
+  if (!except || except.length === 0) {
+    console.warn('Not adding any values to the exceptions will delete everything from the config, '+
```

Comment:
Adding where is this happening exactly 
```suggestion
    console.warn('Utils :: revertDb() :: The 'except' parameter is empty, all documents from the database will be deleted, including Enketo forms from the config, this might cause some automated tests to fail.');
```